### ISSUE TITLE
FEAT: fixed-width fonts in table views

### ISSUE THREAD

**Issue by verygoodsoftwarenotvirus** [beekeeper-studio/beekeeper-studio#968]
just want to say: love ya'll and this software to death. it's great and I recommend it guiltlessly. ya'll are great for making it.

**Is your feature request related to a problem? Please describe.**
Not a problem, just a personal preference. Take the below view, for example:

![image](https://user-images.githubusercontent.com/1289344/148655337-87461d4f-4e99-43f8-bc8a-9ab662a00cbd.png)

IMHO, a fixed-width font would be better for this `id` column, as both of these IDs have the same number of characters. My initial impression when I see this screen is that they don't because of how they appear on top of one another.

**Describe the solution you'd like**
It'd be great to have some way to say "please render the table views with a fixed-width font"

**Describe alternatives you've considered**
There are no theming capabilities in Beekeeper Studio that I'm aware of, so the current alternative I'm using/considering is "dealing with it."

**Comment by rathboma** [comment]
Yes, quality idea. All data tables should have monospaced fonts, I agree.

**Comment by CodeMan99** [comment]
Based on what I see in developer tools, it seems that monospace is already the intention?

Columns like these have the value wrapped in a `<pre>` tag. However, the theme is overriding the `font-family`.

<img width="1904" height="364" alt="Image" src="https://github.com/user-attachments/assets/278b2fca-a02e-478b-b92e-d1b8073164ca" />

**Comment by mcanepa** [comment]
This is what it looks like for a datetime field

<img width="211" height="306" alt="Image" src="https://github.com/user-attachments/assets/7855c983-5e43-4ca5-b3d8-0793c0cf293e" />

compared to vscode

<img width="217" height="209" alt="Image" src="https://github.com/user-attachments/assets/c0092730-3b50-4ca8-89be-4ac759f187cf" />
